frontend/src/container/FormAlertRules/BasicInfo.tsx

```diff
@@ -54,6 +60,11 @@ function BasicInfo({
 		});
 	};
 
+	const noChannels = !channels.payload?.length;
```

Comment:
if the API fails then this would still evaluate to `true` and the tooltip would appear. ( Create a notification channel to configure alert ) , is this expected ?

Reply:
Done